netCDF4/_netCDF4.pyx

```diff
@@ -1421,6 +1508,10 @@ compound types defined for the `netCDF4.Group` or `netCDF4.Dataset` to instances
 variable-length types defined for the `netCDF4.Group` or `netCDF4.Dataset` to instances 
 of the `netCDF4.VLType` class.
 
+**`enumtypes`**: The `enumtypes` dictionary maps the names of
```

Comment:
I guess if you already have these, it makes sense to add `enumtypes` for consistency. That said -- how else would you find out what sort of exotic types are being used other than looking at the types of each variable? That seems like the most straightforward way to do these things.